packages/wallet/ui/README.md

```diff
@@ -24,9 +40,22 @@ open http://localhost:3000/wallet/
 # Should get a Keplr connection prompt when you confirm.
 # Should get "query not found" error, wallet not initialized yet
 
-# Fund keplr
-cd packages/cosmic-swingset
-# Copy the agoric address from your keplr wallet here, starts with "agoric1"
-KEPLR_ADDRESS=<yours>
-make ACCT_ADDR=$KEPLR_ADDRESS AGORIC_POWERS=SMART_WALLET fund-acct provision-acct
-# now refresh the wallet UI and purses should load
+https://psm.inter.trade/
+
+## Testing
+
+export AGORIC_NET=ollinet
+
+# make sure your wallet is provisioned on the net.
+agoric wallet list | grep KEPLR_WALLET_ADDR
+# if it's not in the list, provision it
+https://ollinet.faucet.agoric.net/
```

Comment:
do we want ollinet URLs in our source code? I expect it to be a temporary service.